oligo_designer_toolsuite/database/_oligo_database.py

```diff
@@ -618,46 +632,46 @@ def write_database_to_table(
                         else:
                             entry[property] = str(oligo_property).replace("[[", "[").replace("]]", "]")
                 file_tsv_content.append(entry)
-            file_tsv_content = pd.DataFrame(data=file_tsv_content)
-            file_tsv_content.to_csv(file_table, sep="\t", index=False, mode="a", header=first_entry)
+            file_tsv_content_as_df = pd.DataFrame(data=file_tsv_content)
+            file_tsv_content_as_df.to_csv(file_table, sep="\t", index=False, mode="a", header=first_entry)
             first_entry = False
 
         return file_table
 
     def write_oligosets_to_yaml(
```

Comment:
is it intended to not return the file path anymore? if yes, the documentation needs to be adapted